### PR TITLE
INS-3051: Remove extra fields from index.

### PR DESCRIPTION
**Pull request by andreyromancev**

### CODE REVIEW

File: ledger/object/lifeline.go [hunk 1]
```diff
@@ -54,33 +54,10 @@ func CloneLifeline(idx record.Lifeline) record.Lifeline {
...
-	} else {
-		idx.Delegates = []record.LifelineDelegate{}
-	}
-
 	if idx.EarliestOpenRequest != nil {
 		tmp := *idx.EarliestOpenRequest
 		idx.EarliestOpenRequest = &tmp
 	}
 
-	if idx.PendingPointer != nil {
```
Comment (ruz) [inline ledger/object/lifeline.go]: forgot to clone LatestRequest

File: internal/ledger/artifact/artifact.go [hunk 2]
```diff
@@ -160,35 +159,17 @@ func (m *Scope) activateObject(
 	parent insolar.Reference,
 	memory []byte,
 ) error {
-	parentIdx, err := m.IndexAccessor.ForID(ctx, m.PulseNumber, *parent.Record())
```
Comment (ruz) [inline internal/ledger/artifact/artifact.go]: we kept parent check in artifacts.client... to make sure parent reference is valid and not bogus

Comment (andreyromancev) [reply internal/ledger/artifact/artifact.go]: Genesis doesn't have any checks. It just writes data into storage from config.

Comment (ruz) [reply internal/ledger/artifact/artifact.go]: this if below makes sure that the parent exists. what is wrong with keeping these four lines here?

**Pull request merged**